mail-search.adoc

```diff
@@ -14,12 +14,12 @@ Fields listed here help refine your search further.
 
 . Choose a folder from the **Search-In** field.
 . Typing an email address in the **From** field focuses your search on all emails sent *from* that address.
-. Similarly, typing an email address in the *To* field focuses your search on all emails sent *to* that address.
+. Similarly, typing an email address in the **To** field focuses your search on all emails sent *to* that address.
```

Comment:
FYI:
A single pair of asterisks should work as "bold". We can use a single pair of asterisks in most of bold words/sentences in the user guide.
I don't think it is scope of the PR - Japanese translation.
Reference: https://docs.asciidoctor.org/asciidoc/latest/text/bold/

Reply:
Fixed.